AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Storage/SQLite/ModelSchema+SQLite.swift

```diff
@@ -116,6 +116,8 @@ extension ModelField: SQLColumn {
                 // values we accept, but always output the same format.
                 return value.iso8601Date?.iso8601String
             }
+        case .enum:
+            return binding as? String
```

Comment:
Synced with Daniel on this, opening issue to refactor some of this conversion logic into some converter class to be used to take model field types and values into SQLite and GraphQL components